lib/buildChunkGraph.js

```diff
@@ -327,15 +327,16 @@ const visitModules = (
 			c = cgi.chunkGroup;
 		}
 
-		// 2. We store the Block+Chunk mapping as dependency for the chunk
-		let deps = chunkDependencies.get(chunkGroup);
-		if (!deps) chunkDependencies.set(chunkGroup, (deps = []));
+		// 2. We store the Block + Chunk Group mapping as dependency
+		// for the chunk group which is set in processQueue
+		let deps = chunkGroupDependencies.get(chunkGroup);
+		if (!deps) chunkGroupDependencies.set(chunkGroup, (deps = []));
 		deps.push({
 			block: b,
 			chunkGroup: c
 		});
 
-		// 3. We create/update the chunk group info
+		// 3. We enqueue the cunk group info creation/updating
```

Comment:
Would someone please give some comments why we need to enqueue chunkGroupInfo modification? Why we can't just modify it here without additional queue?